docs/sources/shared/visualizations/axis-options-all.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Axis options
+comments: |
+  Most visualizations with axis options use this file. Some, like state timeline and status history, should use axis-options-size as they only have the size option.
+---
+
+Options under the **Axis** section control how the x- and y-axes are rendered. Some options don't take effect until you click outside of the field option box you're editing. You can also press `Enter`.
+
+| Option                             | Description                                                                                                                                                                                                                                                                                                                                                                      |
+| ---------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Time zone                          | Set the desired time zones to display along the x-axis.                                                                                                                                                                                                                                                                                                                          |
+| [Placement](#placement)            | Select the placement of the y-axis.                                                                                                                                                                                                                                                                                                                                              |
+| Label                              | Set a y-axis text label. If you have more than one y-axis, then you can assign different labels using an override.                                                                                                                                                                                                                                                               |
+| Width                              | Set a fixed width of the axis. By default, Grafana dynamically calculates the width of an axis. By setting the width of the axis, data with different axes types can share the same display proportions. This setting makes it easier for you to compare more than one graph’s worth of data because the axes aren't shifted or stretched within visual proximity to each other. |
```

Comment:
See my note [here](https://github.com/grafana/grafana/pull/98548/files#r1920285150). We probably don't need all of this text.
```suggestion
| Width                              | Set a fixed width for the axis. By default, Grafana dynamically calculates the width of an axis. |
```

Reply:
I only intended to reorganize, not edit except to tweak language for the new use case. The text is what is in the documentation now https://grafana.com/docs/grafana/latest/panels-visualizations/visualizations/time-series/#axis-options